Objects/Converters/ConverterRevit/ConverterRevitShared/AllRevitCategories.cs

```diff
@@ -126,32 +145,45 @@ public IRevitCategoryInfo GetRevitCategoryInfo(string categoryName)
     {
       return categoryInfo;
     }
-
+    
     categoryName = CategoryNameFormatted(categoryName);
-    var revitCategoryInfoCache = revitDocumentAggregateCache.GetOrInitializeWithDefaultFactory<IRevitCategoryInfo>();
 
-    categoryInfo = revitCategoryInfoCache.TryGet(categoryName);
-    if (categoryInfo != null)
+    IRevitObjectCache<IRevitCategoryInfo> revitCategoryInfoCache;
+    if (revitDocumentAggregateCache is not null)
     {
-      return categoryInfo;
-    }
+      revitCategoryInfoCache = revitDocumentAggregateCache.GetOrInitializeWithDefaultFactory<IRevitCategoryInfo>();
 
-    foreach (var info in revitCategoryInfoCache.GetAllObjects())
-    {
-      if (categoryName.IndexOf(info.CategoryName, StringComparison.OrdinalIgnoreCase) >= 0)
+      categoryInfo = revitCategoryInfoCache.TryGet(categoryName);
+      if (categoryInfo != null)
```

Comment:
This makes me a bit supicious... we were previously calling `GetAllObjects` instead of this and I'm not sure the reason why we did this.

Your change seems legit though... just wondering if we're missing something here.

Reply:
This is also my most suspicious place, regardless I believe we should test this with real version too, on both Revit and Dynamo to understand we are having side effect on anything. 🤞